src/Echidna.Tests/Utilities/DbExtensions.cs

```diff
@@ -17,21 +18,26 @@ internal static class DbExtensions
         static db =>
         {
             var credentialDirectory = Path.GetFullPath(Path.Combine(TestContext.CurrentContext.TestDirectory, "..", "..", "..", "credentials"));
+            var dataSource = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? null : "localhost";
 
             switch (db)
             {
                 case Db.SqlServer:
-                    return new SqlConnectionStringBuilder { DataSource = @".\SQLEXPRESS", IntegratedSecurity = true, TrustServerCertificate = true }.ConnectionString;
+                    return TestHelper.IsOnCIPipeline
+                        ? new SqlConnectionStringBuilder { DataSource = dataSource ?? @"(local)\SQL2017", UserID = "sa", Password = "Password12!", TrustServerCertificate = true }.ConnectionString
+                        : new SqlConnectionStringBuilder { DataSource = dataSource ?? @".\SQLEXPRESS", IntegratedSecurity = true, TrustServerCertificate = true }.ConnectionString;
                 case Db.SystemDataSqlServer:
-                    return new SqlConnectionStringBuilder { DataSource = @".\SQLEXPRESS", IntegratedSecurity = true }.ConnectionString;
+                    return TestHelper.IsOnCIPipeline
+                        ? new SqlConnectionStringBuilder { DataSource = dataSource ?? @"(local)\SQL2017", UserID = "sa", Password = "Password12!" }.ConnectionString
+                        : new SqlConnectionStringBuilder { DataSource = dataSource ?? @".\SQLEXPRESS", IntegratedSecurity = true }.ConnectionString;
                 case Db.Postgres:
                     {
-                        var (username, password) = ReadCredentials("postgres.txt");
-                        return new NpgsqlConnectionStringBuilder { Port = 5433, Host = "localhost", Database = "postgres", Username = username, Password = password }.ConnectionString;
+                        var (username, password) = ReadCredentials(TestHelper.IsOnCIPipeline ? "postgres.ci.txt" : "postgres.txt");
+                        return new NpgsqlConnectionStringBuilder { Port = 5432, Host = "localhost", Database = "postgres", Username = username, Password = password }.ConnectionString;
```

Comment:
This is the default port for Postgres, so AppVeyor also uses 5432 by default. I'd like to use the default port unless there's a good reason not to.